### DISCUSSION TITLE
429 Too Many Requests 错误的解决方法

### DISCUSSION THREAD

**Discussion by Everless321** [messense/aliyundrive-webdav#128] [Q&A]
登录网页版 解决方法登录网页版 按f12打开控制台—application—local storage 清理掉现存的数据 然后刷新重新获取数据 重新填入新的token

我的机器上使用过之后是有效的 建议遇到这个错误的也都试一下这个方法

**Comment by messense** [comment]
👍

非 bug 报告请在讨论区（Discussions）中开贴哦。

**Reply by heavenkiller2018** [reply]
按上面试了，还是没有效果啊？还是显示 429 Too Many Requests

**Reply by messense** [reply]
有没有同一个账号使用其他类似的软件？重新登录网页版重新获取 refresh token 试试？

我的账号只使用这个服务的情况下，目前稳定运行了 15 天。

```bash
root@nanopi:~# uptime
 18:07:58 up 15 days,  7:45,  load average: 0.01, 0.02, 0.00
```
